app/package.json

```diff
@@ -21,6 +21,7 @@
     "dayjs": "^1.11.18"
   },
   "devDependencies": {
+    "@rollup/rollup-win32-x64-msvc": "^4.52.4",
```

Comment:
@Randyseriosa this will break cloudflare deployment

checkout app/scripts and on how it works with postinstall